data/hai/unfettered 1 war preparations.txt

```diff
@@ -0,0 +1,1229 @@
+				accept
+	waypoint "Hi Yahr"
+	on waypoint
+		dialog `You indeed notice the traces as you come back to the system where you found the Auxiliary. You figure they probably went east next with their jump drive, avoiding Hai territory.`
+	on enter "Celeborim"
+		dialog `You are on the right track. There are small traces here as well.`
+	on enter "Asikafarnut"
+		dialog `You are still on the right path, and you even find some small debris of ships. They probably had to stay here a bit longer to fight them off.`
+	on enter "Makferuti" "Sobarati"
+		dialog `There are no traces here. They probably went to another system, looking for refuge from the hostile drones.`
+	on enter "Silikatakfar" "Eneremprukt"
+		dialog `You cannot find any traces here. This is not the right direction.`
+	on enter "Mesuket"
+		dialog `This system looks dangerous. There is too much interference from the debris to pick up the traces, but you figure they would try to avoid this system.`
+	on enter "Persitar"
+		dialog `This doesn't look like a good hiding spot...`
+	on enter "Ferukistek"
+		dialog `There are traces here indeed. You spot one system with no hyperspace links north-east of here. It seems like a logical place for the Alphas to go, considering the drones do not appear to have jump drives.`
+	on enter "Host"
+		dialog `You arrive in the system expecting to find a vault and the ship repairing, but it's a trap! The Auxiliary from earlier is indeed here, but it's not damaged at all, and it looks like it's waiting for you with quite a few drones. Whatever the case, it probably has the jump drives on board or on the nearby planet. You should first capture the ship, as asked by the Unfettered, and then inspect the planet. Strangely, you do not find a single trace of the particles in this system.`
+	
+	npc capture
+		government "Alpha"
+		system "Host"
+		personality heroic nemesis staying vindictive target
+		ship "Auxiliary (Alpha)" "Settler"
+		conversation
+			action
+				set "unfettered: capped auxiliary"
+			`While capturing the ship, you noticed that only very few of the crew were actually Alphas. What little were Alphas seem to have disappeared. They probably used an escape pod to get away unoticed in the middle of the fight, or maybe they were never there in the first place. Whatever the case, you cannot find any trace of them. The crew was composed of mercenaries. There was no doubt who was in command of this vessel, however. The jump drives don't seem to be aboard the ship. You should head to the Alpha base to see if you can find them or anything else of interest.`
```

Comment:
Do you remember where it was? I couldnt find it
They're on your ship when you come back to the UHai, because you picked them up at the base, but I didnt see the other place